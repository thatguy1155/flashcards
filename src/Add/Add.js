import React from 'react';

export default function Add() {
  return (
    <div className="add-page">
      <p>this is the add page</p>
    </div>
  );
}
