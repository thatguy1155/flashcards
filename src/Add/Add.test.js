import TestRenderer from 'react-test-renderer';
import Add from './Add';

describe('Header', () => {
  test('non-shallow render', () => {
    // eslint-disable-next-line new-cap
    const element = new TestRenderer.create(
      <Add />,
    );
    expect(element.toJSON()).toMatchSnapshot();
  });
});
