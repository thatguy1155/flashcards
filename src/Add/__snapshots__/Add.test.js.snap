// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header non-shallow render 1`] = `
<div
  className="add-page"
>
  <p>
    this is the add page
  </p>
</div>
`;
